README.md

```diff
@@ -2,152 +2,344 @@
 
 ## Module Control.Alt
 
-### Type Classes
+#### `Alt`
 
-    class (Functor f) <= Alt f where
-      (<|>) :: forall a. f a -> f a -> f a
+``` purescript
+class (Functor f) <= Alt f where
+  (<|>) :: forall a. f a -> f a -> f a
+```
+
+The `Alt` type class identifies an associative operation on a type.
```

Comment:
I wouldn't even call things with kind `# *` types. It's a bit confusing because kinds _classify_ types, so type constructors are types too right? But I like to use the terms type, type constructor and row, to try to make things clear for beginners. Maybe it doesn't help, I'm not sure..

This looks great otherwise by the way. We can keep this as is, I'm just being picky.